src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNITcpHandle.cs

```diff
@@ -223,6 +223,14 @@ void Cancel()
                     if (ipAddresses[i] != null)
                     {
                         sockets[i] = new Socket(ipAddresses[i].AddressFamily, SocketType.Stream, ProtocolType.Tcp);
+#if netcoreapp
+                        // enable keep-alive on socket
+                        const int time = 30;
+                        const int interval = 1000;
```

Comment:
These probably don't need to be constants and can just be passed directly into the method calls... the names here don't really add anything.  If we do keep the consts, though, they should be PascalCased.

Reply:
done